clients/java/dkv-client/src/main/java/org/dkv/client/DKVClient.java

```diff
@@ -280,6 +280,12 @@
      */
     Iterator<DKVEntry> iterate(byte[] startKey, byte[] keyPref);
 
+    /**
+     * This methods returns the approximate count of keys in the DKV database
+     * @return retunrn the count of keys
+     */
+    long getDbSize();
```

Comment:
We don't  need to add `get` as a prefix as its implicit that `dBSize` is a read-only operation